src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -3138,6 +3141,9 @@ SpinelNCPInstance::regsiter_all_set_handlers(void)
 	register_set_handler_spinel(
 		kWPANTUNDProperty_ThreadRouterDowngradeThreshold,
 		SPINEL_PROP_THREAD_ROUTER_DOWNGRADE_THRESHOLD, SPINEL_DATATYPE_UINT8_C);
+	register_set_handler_spinel(
+		kWPANTUNDProperty_NCPCoexEnable,
+		SPINEL_PROP_RADIO_COEX_ENABLE, SPINEL_DATATYPE_BOOL_C);
```

Comment:
Two suggestions: 
-  I suggested we add a spinel cap for COEX, which we can then use here.
- I also think we want to persist this property in wpantund (if the user sets it and afterwards NCP gets reset, we remember the value in `wpantund` and set it back on NCP during its initialization).

To change this to persist and perform check capability, we just need to use the method `register_set_handler_capability_spinel_persist()` instead to register the property

Reply:
Have updated.